js/ui/components/clubhouse.js

```diff
@@ -60,7 +57,7 @@ const ClubhouseIface =
   </interface> \
 </node>';
 
-function _clipToMonitor(actor) {
+function clipToMonitor(actor) {
```

Comment:
~~Is there any advantage by renaming it? I would keep it as it was before.~~

Reply:
it's used in oldClubhouse.